assets/js/src/core/components/tag/tag.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { Tag as AntTag, type TagProps as AntTagPropsProps } from 'antd'
+import React from 'react'
+import { useStyles } from './tag.styles'
+import { Icon } from '@Pimcore/components/icon/icon'
+import { type LiteralUnion } from 'antd/es/_util/type'
+import type { PresetColorType, PresetStatusColorType } from 'antd/es/_util/colors'
+
+export interface TagProps extends AntTagPropsProps {
+  text: string
+  iconName?: IconNameType
+  className?: string
+  theme?: TagTheme
+}
+
+export type IconNameType = 'world' | 'user'
+
+export type TagTheme = 'transparent' | 'user-role' | 'admin-role' | 'link-blue' | 'link-purple'
```

Comment:
hmm, for me user-role and admin-role do not really sound like themes. 
I would rather see them as own components, that will implement the tag.

Same for link-blue and link-purple. Mixing the names of the color in here, could create conflicts in the future, when we switch to other colors. And again we are not really adding new functionality, so maybe we can directly use the color and bordered property for it? But we also could think of adding a LinkTag-Component, when you think it would be helpful.

Reply:
Have improved this. 
Let me know what you think about the way I listed the different variations in Storybook.